docs/workflow/requirements/linux-requirements.md

```diff
@@ -47,15 +47,14 @@ Install the following packages for the toolchain:
 * liblttng-ust-dev
 * libssl-dev
 * libkrb5-dev
-* zlib1g-dev
```

Comment:
If somebody follows these instructions to setup their environment, they will again get the build break about missing system zlib.

I think you want to split this PR into two: Deleting zlib zlib-intel sources that can be merged now, and deleting the system zlib from the dev environment that can merged only once we update the SDK used to build to P7.

Reply:
Removed here.

Reply:
Here's the follow-up PR: https://github.com/dotnet/runtime/pull/105419